src/openforms/submissions/models/submission.py

```diff
@@ -702,13 +702,24 @@ def calculate_price(self, save=True) -> None:
         """
         Calculate and save the price of this particular submission.
         """
+        logger.debug("Calculating submission %s price", self.uuid)
```

Comment:
Maybe move this logging call after `if not self.payment_required`?

Reply:
IMO it makes sense to see the following log lines:

```
DEBUG Calculating submission xyz price
DEBUG Submission xyz does not require payment, skipping price calculation
```
or if payment is required:

```
DEBUG Calculating submission xyz price
```
I guess it provides a consistent "anchor" point to grep for in the logs, and possibly there is extra information?